include/linux/page_compat.h

```diff
@@ -1,6 +1,8 @@
 #ifndef	_ZFS_PAGE_COMPAT_H
 #define	_ZFS_PAGE_COMPAT_H
 
+#include <linux/version.h>
```

Comment:
Should be dropped when refreshed to include the configure check

Reply:
Dropped accordingly.